metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/api/internal/RecordParser.java

```diff
@@ -17,6 +17,14 @@
    */
   Set<SearchTermContext> parseSearchTerms(RDF rdf);
 
+  /**
+   * This method parses records to obtain search terms for enrichment, related to aggregation.
+   *
+   * @param rdf The record.
+   * @return The search terms. It is recommended to return one object per term-language combination.
+   */
+  Set<SearchTermAggregation> parseAggregationSearchTerms(RDF rdf);
```

Comment:
The end result should be, among other things, that this is no longer needed as a separate method.